Applications/spire/source/book_view_ui_tester/random_book_view_model.cpp

```diff
@@ -132,7 +132,25 @@ connection RandomBookViewModel::connect_volume_slot(
   return m_volume_signal.connect(slot);
 }
 
-void RandomBookViewModel::update() {}
+void RandomBookViewModel::update() {
+  auto random_num = m_random_engine() % 3;
+  if(random_num == 0) {
+    return;
+  }
+  auto& bid_price = m_bbo.m_bid.m_price;
+  auto& ask_price = m_bbo.m_ask.m_price;
+  if(random_num == 1) {
+    if(boost::float64_t(bid_price) > 0.01) {
```

Comment:
Do not use floating points. Use Money::CENT to represent 1/100th of a dollar. If you need 10 cents you do 10 * Money::CENT. If you need units, then use Money::ONE. You can look up all the predefined units in the Money.hpp header file.